tslint.json

```diff
@@ -56,7 +53,7 @@
     "trailing-comma": [
       true,
       {
-        "multiline": "always",
```

Comment:
this is something that I would expect to see handled in a separate issue on a different branch, not here in making unit tests for the shortcut service.  Can we revert and make a diff issue for this?